_data/hosting_providers.json

```diff
@@ -2259,4 +2259,14 @@
     "reviewed": "2020.01.07",
     "note": "Available for all plans from free account to dedicated servers"
   }
+  {
+    "name": "Wikstrom Telephone Company (Wiktel)",
+    "link": "https://wiktel.com",
+    "category": "full",
+    "tutorial": "",
+    "announcement": "",
```

Comment:
Any chance you have an announcement page that we can link users to that states that there are no additional steps required to set up an SSL certificate when creating a server? If not, could you please confirm here that this is the case?

Reply:
We do not have an announcement page. The best I can offer is this: https://wiktel.com/webhosting

TLS is automatically configured on all sites and does not require customer action. We use Let's Encrypt on our "Economy" package (which, despite the marketing name, is something like 80+% of our web hosting). On our cPanel server, we use cPanel's certificate system, which chains from Comodo.

Reply:
Ok, in that case I'd suggest just adding a short message explaining this to the "note" field.